### ISSUE TITLE
Suggestions for developing the "Fields manager for Checkout Billing & Shipping forms" feature

### ISSUE THREAD

**Issue by 79ho3ein** [CrocoBlock/suggestions#3793]
hello
There are suggestions for developing this feature. Please pay attention to them, because they are very important.

1-This feature has been added to the plugin in the best possible way. **But it lacks a very important case.**
That we can only style text fields and customize them, And we can not customize list fields like province or city. 

Please see the image below :
![Screenshot (112)](https://user-images.githubusercontent.com/77163248/128575942-6e50d734-9d5e-4861-ae45-45a99de496c1.png)
As you can see in the image above, text fields such as contact number, email, address, name, etc. (text fields) are the same. And there is no problem with personalizing them. But in this update, no thought has been given to list fields such as city and province. And we can not personalize them.
**Please add settings so that we can customize all the fields in the list.** See a full example of this request in the image below : 
![Screenshot (113)](https://user-images.githubusercontent.com/77163248/128576413-9c69375f-01dc-4eaa-be4f-87f962bb11c0.png)
In the image above, both the text field and the list are designed in the same style, and rightly so.
**In some countries, both the city and country fields are listed. Please consider this as well.**

2- Please add settings so that we can customize the contents of the list fields.The image below is a perfect example of this offer :
![Screenshot (114)](https://user-images.githubusercontent.com/77163248/128577011-05f390e7-3d26-4721-915e-3f5e6ad9d976.png)

Items that need personalization : 
1- All boders in the list. (green color)
2- Customize the search box (red color)
3- Typography settings (for city and country names)
4- Apply background color for active mode (black color)
5- Apply background color for hover mode (green color)

See a complete example of this request in the image below : 
![Screenshot (115)](https://user-images.githubusercontent.com/77163248/128577627-02dea210-296d-4a03-bea1-5b377c679aec.png)

**Please prioritize these suggestions if possible. The "Fields manager for Checkout Billing & Shipping forms" feature is great. Needs more action to complete.**

**It should be noted that the above requests are styles that are observed in all sites and templates and the request is not outside the framework.**

thank you🌹

**Comment by UraraReika** [comment]
We test it on theme Kava with enabled manage fields option and everything seems to be working correctly. Perhaps this is related to your site. Please contact our support team. Thank you!

**Comment by 79ho3ein** [comment]
Thank you @UraraReika 
Yes, there is no problem in ready theme ...
But Unfortunately, "Fields manager" **is not compatible with the hello elementor theme**.  And that's why list fields have a problem.The image I posted above was related to the hello elementor theme.
It would be great if you could add a feature that allows us to customize the list fields.  So we will no longer have such a problem with the hello elementor theme.

Thank you

**Comment by 79ho3ein** [comment]
In the second case (editing the list and active and inactive mode) there is a problem in hello elementor.  Because there are no settings to personalize the contents of the list field. And everything is by default.
**In general, there is no compatibility with the hello elementor theme**

**Comment by UraraReika** [comment]
what do you mean by personalization?

**Comment by 79ho3ein** [comment]
excuse me... I meant Customization.

Because, as I said above, customizing text fields is different from listing fields such as country, city, or province. It would be great if you could add settings so that we could completely customize the list fields.

**The second very important thing that is necessary in customizing list fields is the ability to edit the contents of the list fields.**
These parts : https://prnt.sc/1log2n4
Now I fixed it with css but it is not interesting. It will definitely be better in the Woobuilder jet.

Thank you for your efforts. You are really unique.

**Comment by Michaelgimii** [comment]
Yes, exactly ! I also use the hello elementor theme. Unfortunately, the selected fields do not look good.
We need to be able to customize the selected fields.
Please add it in the next version. I and many users are not coders and we need this feature. **Customize select fields such as city, province or country**Thanks to the crocoblock team

**Comment by 79ho3ein** [comment]
hello @UraraReika , Will compatibility with hello theme be added?

**Comment by Michaelgimii** [comment]
Is there any news about this feature?

**Comment by Michaelgimii** [comment]
hi @UraraReika , Sorry, is there any news about this feature? Is it clear in which version it will be published? This update will complete the checkout page customization. We are very excited to release it.

**Comment by Joliamendi** [comment]
Great crocoblock team, do you have any plans to release this feature? It has been almost a year since this request was filed. Even a news that you are working on it will be very happy.

**Comment by Michaelgimii** [comment]
Hi @UraraReika.
Almost two years have passed since this feature was requested. Is there any news about it? Even if the answer is no, I hope you will answer. Thankful

**Comment by aalexandr21** [comment]
Unfortunately, we do not plan to introduce such functionality into our plugin. Now our team is working on more priority tasks. However, in the future, we will not exclude the possibility of adding this functionality to the core of our plugin.

Thank you for understanding.
